src/style.css

```diff
@@ -0,0 +1,117 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+.for {
+  margin-top: 20px;
+}
+
+.input {
+  font-family: Avenir, sans-serif;
+  border: 1px;
+  width: 100%;
+  border-radius: 4px;
+  outline: none;
+  padding: 23px 62px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  font-weight: 900;
+  color: #3d4e61;
+}
+
+.body {
+  margin: 0;
+  padding: 0;
+}
+
+.bigger {
+  height: 70px;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  background: url(images/Search.svg) top 25px left 26px;
+  background-repeat: no-repeat;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+  background-size: 19px;
+}
+
+.smaller {
+  height: 42px;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
```

Comment:
A similar style with big input. You can add this for input class and don't repeat